deck.go

```diff
@@ -70,6 +72,33 @@ func LoadDeck(dev *streamdeck.Device, base string, deck string) (*Deck, error) {
 		d.Widgets = append(d.Widgets, w)
 	}
 
+        watcher, err := fsnotify.NewWatcher()
+        err = watcher.Add(path)
+
+        go func() {
+		for {
+			select {
+				case event := <-watcher.Events:
+					if currentDeck == path {
+						fmt.Println("Change:  %s: %s", event.Op, event.Name)
```

Comment:
Careful, `Println` doesn't expect formatting directives like `%s`. You'll have to use `Printf` here and finish it with a `\n` for a newline.